src/auditlog/models.py

```diff
@@ -3,15 +3,18 @@
 import json
 import ast
 
+from six import python_2_unicode_compatible
+from six import iteritems, integer_types
+
```

Comment:
These imports could be a bit neater

Reply:
I will fix this once I get on my computer

Reply:
Unnecessary blank lines removed and imports have been formatted properly